docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -79,33 +79,102 @@ features, like:
 Note that data modification also requires implementing
 a :ref:`connector-page-sink-provider`.
 
+When a ``SELECT`` query is being executed, and it contains specific clauses,
+the engine can try pushing down some of them into the connector, by calling
+one of the following methods of the ``ConnectorMetadata`` service:
+
+* ``applyLimit`` for the ``LIMIT`` clause, or ``applyTopN`` when ``ORDER BY``
+  is also present
```

Comment:
nit

```suggestion
* ``applyLimit`` for the ``LIMIT`` clause, or ``applyTopN`` when both ``LIMIT`` and ``ORDER BY``
  clauses are present
```

Reply:
topN would be called when only `ORDER BY` is present